gammapy/datasets/make.py

```diff
@@ -64,3 +70,90 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def generate_observation_table(observatory, n_obs):
+    """Generate an observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory : string
+    	name of the observatory
+    n_obs : integer
+    	number of observations for the obs table
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    astro_table = Table()
+
+    # obs id
+    obs_id = np.arange(n_obs_start, n_obs_start + n_obs)
+    col_obs_id = Column(name='OBS_ID', data=obs_id)
+    astro_table.add_column(col_obs_id)
+
+    # on time
+    ontime = Quantity(30.*np.ones_like(col_obs_id.data), 'minute')
+    col_ontime = Column(name='ONTIME', data=ontime)
+    astro_table.add_column(col_ontime)
+
+    # livetime
+    livetime = Quantity(25.*np.ones_like(col_obs_id.data), 'minute')
+    col_livetime = Column(name='LIVETIME', data=livetime)
+    astro_table.add_column(col_livetime)
+
+    # TODO: add columns for coordinates:
+    #       pointing observation -> done
+    #       and date -> done
+    #       then convert to alt az, considering the observatory location
+    # TODO: add column(s) for offset, and take it into account in the coord. transformation!!!
+
+    # RA, Dec
+    # random points on a sphere ref: http://mathworld.wolfram.com/SpherePointPicking.html
```

Comment:
There's a utility function for this already: https://gammapy.readthedocs.org/en/latest/api/gammapy.utils.random.sample_sphere.html
Should be possible to re-use it here?